includes/settings/settings-page.php

```diff
@@ -241,6 +241,10 @@ final public function register_settings_fields() {
 
 				add_settings_section( $full_section_id, $label, $section_callback, static::PAGE_ID );
 
+				if ( 'experiments' === $section_id ) {
+					add_settings_section( 'elementor-experiments-ongoing-section', $label, $section_callback, static::PAGE_ID );
+				}
+
```

Comment:
Isn't it a generic page that should get its content from a dedicated file? (a file that is dedicated to the content of the experiment?).

Reply:
This is how WordPress works, I don't think we can add setting fields in a different way.
@kobizz What do you think?